app/scenes/Search/components/UserFilter.tsx

```diff
@@ -24,11 +25,13 @@ function UserFilter(props: Props) {
     const userOptions = users.all.map((user) => ({
       key: user.id,
       label: user.name,
+      icon: <Avatar model={user} />,
     }));
     return [
       {
         key: "",
         label: t("Any author"),
+        icon: <Avatar />,
```

Comment:
@tommoor right now it's showing a hollow ring. Should we change it to something else?

Reply:
How about `<UserIcon />` ?